docs/source/quickstart.rst

```diff
@@ -8,6 +8,15 @@ Install
 
     $ pip install distributed --upgrade
 
+**Note for Macports users:** There `is a known issue <https://trac.macports.org/ticket/50058>`_.
+with python from macports that makes executables be placed in a location that is not
+available by default. A simple solution is to extend the `PATH` environment
+variable to the location where python from macports install the binaries:
+
+```
+$ export PATH=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin:$PATH
+```
+
```

Comment:
Sure.  Although I don't think we need to be as involved.  Hopefully writing
this only takes up a small amount of time.

On Fri, Feb 12, 2016 at 11:57 AM, Daniel Rodriguez <notifications@github.com
> wrote:

> In docs/source/quickstart.rst
> <https://github.com/dask/distributed/pull/132#discussion_r52790693>:
>
> > @@ -8,6 +8,15 @@ Install
> >
> >      $ pip install distributed --upgrade
> >
> > +**Note for Macports users:** There `is a known issue <https://trac.macports.org/ticket/50058>`_.
> > +with python from macports that makes executables be placed in a location that is not
> > +available by default. A simple solution is to extend the `PATH` environment
> > +variable to the location where python from macports install the binaries:
> > +
> > +```
> > +$ export PATH=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin:$PATH
> > +```
> > +
>
> If you want to do that route I always liked the one for cryptograpy:
> https://cryptography.io/en/latest/installation/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dask/distributed/pull/132/files#r52790693>.
>